apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -236,6 +240,17 @@ type OutputTLSSpec struct {
 	//
 	// This option is *not* recommended for production configurations.
 	InsecureSkipVerify bool `json:"insecureSkipVerify,omitempty"`
+
+	// TLSSecurityProfile is the security profile to apply to the output connection
+	TLSSecurityProfile *TLSSecurityProfileSpec `json:"securityProfile,omitempty"`
+}
+
+// TLSSecurityProfileSpec sets the TLS security profile.
+type TLSSecurityProfileSpec struct {
```

Comment:
@jcantrill We don't need a new type here, config/v1/TLSSecurityProfile will do just fine

Reply:
I would rather name it "tlsSecurityProfile", to keep consistent with https://github.com/openshift/api/blob/master/config/v1/types_apiserver.go#L56

Reply:
@jcantrill We don't want a new type here, we actually need tlsSecurityProfile to be config/v1/TLSSecurityProfile https://github.com/openshift/api/blob/master/config/v1/types_tlssecurityprofile.go#L6. It's a bit more involved than just the name of a profile.